common/hexutil/json.go

```diff
@@ -234,6 +268,21 @@ func (b Uint64) String() string {
 	return EncodeUint64(uint64(b))
 }
 
+func (b Uint64) ImplementsGraphQLType(name string) bool { return name == "Long" }
+
+func (b *Uint64) UnmarshalGraphQL(input interface{}) error {
+	var err error
+	switch input := input.(type) {
+	case string:
+		return b.UnmarshalText([]byte(input))
+	case int32:
+		*b = Uint64(input)
+	default:
+		err = fmt.Errorf("Unexpected type for BigInt: %v", input)
```

Comment:
```suggestion
		err = fmt.Errorf("Unexpected type for Uint64: %v", input)
```

Reply:
Or `Long` ?

Reply:
You're right, it should be `Long`. Fixed.